File: AndroidStudioProjects/JavaOOPDetails/app/src/main/java/com/ecrinaybay/javaoopdetails/MainActivity.java
package com.ecrinaybay.javaoopdetails;

import static com.ecrinaybay.javaoopdetails.Kedi.turIsmi;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Kedi kedi = new Kedi("Kedi1","Sari","Kahverangi");
        Kedi kedi2 = new Kedi();
        kedi2.isim = "Kedi 2";
        System.out.println(kedi2.isim);
        System.out.println(kedi.turIsmi);
        kedi.konusKedi(); //Static olmayanlara objenin kendisinden ulaştık.
        Kedi.konusKediStatic(); //Staticlere sınıftan ulaşırız.

        Kopek kopek = new Kopek();
        Kopek kopek2 = new Kopek("Barley","Sari" ,"Sari");
        System.out.println(kopek2.tuyRengi);

        Hayvan hayvan = new Hayvan();

    }
    public void test(){

    }

}

File: AndroidStudioProjects/ArtBookTesting/app/src/main/java/com/ecrinaybay/artbooktesting/view/MainActivity.kt
package com.ecrinaybay.artbooktesting.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Window
import android.view.WindowManager
import com.ecrinaybay.artbooktesting.R
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {

    @Inject
    lateinit var fragmentFactory: ArtFragmentFactory

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportFragmentManager.fragmentFactory = fragmentFactory
        setContentView(R.layout.activity_main)
        window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN)
    }
}


















































